Makefile

```diff
@@ -25,6 +25,7 @@ implementations := \
 	knapsack \
 	raindrops \
 	phone-number \
+	bob \
```

Comment:
I don't know if this is where you actually want `bob` in this list.  It is where it would have been when I first opened this PR.